Taylor_series_Repo/Taylor_Series_Horner_Rule.java

```diff
@@ -0,0 +1,26 @@
+// Taylor series using Horner Rule in Java
+
+import java.util.Scanner;
+public class Test {
+    static double s; // Declare static variable
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int x = sc.nextInt();
+        int num = sc.nextInt();
+        double result = exponential(x, num);
+        System.out.println(+result);
+    }
+
+    //Declare the function for finding exponential

```

Comment:
Always add a space after **//** in comments. Rest of the file looks good. :sparkles: